generators/client/templates/angular/src/main/webapp/app/shared/auth/_principal.service.ts

```diff
@@ -9,7 +9,7 @@ import { <%=jhiPrefixCapitalized%>TrackerService } from '../tracker/tracker.serv
 @Injectable()
 export class Principal {
     private _identity: any;
-    private authenticated = false;
+    private authenticated: boolean = false;
```

Comment:
type not required as its inferred from the value and this will also create a tslint warning